previews/primer/alpha/banner_preview.rb

```diff
@@ -24,6 +23,16 @@ def default
         render(Primer::Alpha::Banner.new) { "This is a banner." }
       end
 
+      # @label Announce
+      def announce_on_show
+        render_with_template
+      end
+
+      # @label Announce when updating wrapper visibility
+      def announce_on_show_with_hidden_wrapper
+        render_with_template
+      end
+
```

Comment:
I want to add a test for when `Primer::Alpha::Banner` is dynamically injected. How should I set this up?